gcloud/_helpers.py

```diff
@@ -30,9 +30,13 @@
 except ImportError:
     app_identity = None
 try:
-    from grpc.beta import implementations
+    from google.gax.grpc import exc_to_code as beta_exc_to_code
+    import grpc
+    from grpc._channel import _Rendezvous
```

Comment:
Right, so the intention is that you will also make use of your knowledge that [the caught `grpc.RpcError` instance is also a `grpc.Call`](https://github.com/grpc/grpc/blob/c41f81aeeb054ec436934c7cc1afeb1dee0a8581/src/python/grpcio/grpc/__init__.py#L473).